frontends/p4/validateStringAnnotations.h

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2019 VMware, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#ifndef P4_VALIDATESTRINGANNOTATIONS_H_
```

Comment:
Strange that this works, shouldn't it be `FRONTENDS_P4_VALIDATESTRINGANNOTATIONS_H_`?

Reply:
We probably don't have a linter check for it or it allows a suffix :-/